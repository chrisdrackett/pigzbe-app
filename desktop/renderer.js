import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
