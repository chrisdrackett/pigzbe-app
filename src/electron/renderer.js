import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
