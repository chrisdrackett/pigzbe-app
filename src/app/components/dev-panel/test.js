import React from 'react';
import renderer from 'react-test-renderer';
import DevPanel from './';

describe('DevPanel', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<DevPanel/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
