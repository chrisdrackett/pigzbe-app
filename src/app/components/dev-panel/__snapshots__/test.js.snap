// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DevPanel renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "left": 0,
      "paddingLeft": "9.375%",
      "paddingRight": "9.375%",
      "position": "absolute",
      "top": 30,
      "width": "100%",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "backgroundColor": "rgb(149, 198, 100)",
          "color": "rgb(255, 255, 255)",
          "fontFamily": "Poppins",
          "fontSize": 12,
          "fontWeight": "bold",
          "paddingLeft": 4,
          "paddingRight": 4,
        },
        Object {
          "backgroundColor": "rgb(255, 0, 0)",
        },
      ]
    }
  >
    LIVENET
  </Text>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "height": 18,
        "opacity": 1,
        "width": 18,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/app/components/header/images/settings-icon.png",
        }
      }
      style={
        Object {
          "height": 18,
          "width": 18,
        }
      }
    />
  </View>
</View>
`;
